File: 2.0/java/media/findMedia.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.callstexts.model.Media;
import com.callfire.api.client.api.callstexts.model.request.FindMediaRequest;
import com.callfire.api.client.api.common.model.Page;

class ApiClientSample {
    public static void main(String[] args) throws ParseException {
        CallfireClient client = new CallfireClient("api_login", "api_password");

        FindMediaRequest request = FindMediaRequest.create()
                .filter("image1.jpeg")
                .build();

        Page<Media> media = client.mediaApi().find(request);
        // check media.getItems() list for stored data
    }
}

File: 2.0/csharp/media/findMedia.cs
using System;
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model;
using CallfireApiClient.Api.CallsTexts.Model.Request;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var request = new FindMediaRequest
        {
            Filter = "image1.jpeg"
        };

        Page<Media> media = client.MediaApi.Find(request);
        // check media.Items for stored data
    }
}